### PR TITLE
Noted MySQL 5.7 requirement and blocked WSL.

### PR DESCRIPTION
**Pull request by bencoman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,3 +87,16 @@ user_track_ids.each do |id|
...
+The following error is seen using MySQL prior to version 5.7 a discussed at gogs/gogs#4894.
+```
+$ bundle exec rake exercism:setup
+Mysql2::Error: Specified key was too long; max key length is 767 bytes:
+```
+If your system doesn't have a pre-built 5.7 package, [this install log]() may be helpful.
+
+### Windows Subsystem For Linux
+Installation on Windows Subsystem For Linux requires Windows Version 1809 (release due October 2018). 
+Tracked at #4346.
```
Comment (iHiD) [inline README.md]: Could you link to the right Repo here pls?

File: README.md [hunk 2]
```diff
@@ -87,3 +87,16 @@ user_track_ids.each do |id|
...
 ```
+
+## Troubleshooting
+### MySQL < 5.7
+The following error is seen using MySQL prior to version 5.7 a discussed at gogs/gogs#4894.
+```
+$ bundle exec rake exercism:setup
+Mysql2::Error: Specified key was too long; max key length is 767 bytes:
+```
+If your system doesn't have a pre-built 5.7 package, [this install log]() may be helpful.
```
Comment (iHiD) [inline README.md]: This link is missing

File: README.md (hunk 1 above)
Comment (bencoman) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (bencoman) [reply README.md]: fixed

**Comment by iHiD** [comment]
Brilliant. Thanks :)

**Comment by bencoman** [comment]
@iHiD I notice the cross repo issue references I added are not live links like I expected.

**Comment by iHiD** [comment]
Ah. Good catch. Mind putting up another PR for it. If not, I'll do it when I get chance.

**Comment by bencoman** [comment]
I'll do it.  Intuitively I thought issue auto-linking would work here.  
In hindsight README is a repo file, outside the Github website.

**Pull request merged**